TicTacToe.java

```diff
@@ -139,6 +145,8 @@ private static void initialize(Scanner input) {
 			}
 			catch(NumberFormatException nfe) {}
 		}
```

Comment:
This error catching can be much cleaner. You have a loop that runs while `true` but you can run it while the input is not `1-4` and currently you are breaking out of it but you need to properly handle the error

Reply:
So what would be a better way?